DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfigurationController.cs

```diff
@@ -116,5 +119,114 @@ public IActionResult EditCentreWebsiteDetails(EditCentreWebsiteDetailsViewModel
 
             return RedirectToAction("Index");
         }
+
+        [HttpGet]
+        [Route("RegistrationPrompts/{promptNumber}/Edit")]
+        public IActionResult EditRegistrationPrompt(int promptNumber)
+        {
+            var centreId = User.GetCentreId();
+
+            var customPrompt = customPromptsService.GetCustomPromptsForCentreByCentreId(centreId).CustomPrompts
+                .Single(cp => cp.CustomPromptNumber == promptNumber);
+
+            return View(new EditRegistrationPromptViewModel(customPrompt));
+        }
+
+        [HttpPost]
+        [Route("RegistrationPrompts/{promptNumber}/Edit")]
+        public IActionResult EditRegistrationPrompt(EditRegistrationPromptViewModel model, string action)
+        {
+            if (action.StartsWith("delete") && TryGetAnswerIndexFromAction(action, out var index))
+            {
+                return EditRegistrationPromptPostRemovePrompt(model, index);
+            }
+
+            return action switch
+            {
+                "save" => EditRegistrationPromptPostSave(model),
+                "addPrompt" => EditRegistrationPromptPostAddPrompt(model),
+                _ => RedirectToAction("Error", "LearningSolutions")

```

Comment:
This is great - do you mind adding this to the Edit details switch statement too? 🙏